### ISSUE TITLE
Tailscale SSH problem 

### ISSUE THREAD

**Issue by jim13x** [iamromulan/quectel-rgmii-configuration-notes#8]
I was unable to start an SSH session over tailscale, and I could see 'ILLEGAL ROOT LOGIN  on '/dev/pts/4' from ' in the logs.  I found this post:

[ILLEGAL ROOT LOGIN  on '/dev/pts/4' from ](https://community.victronenergy.com/questions/252085/illegal-root-login-and-available-ttys-for-ssh.html)

The following fixed it:

echo 'pts/4
pts/5
pts/6
pts/7' >> /etc/securetty

**Comment by iamromulan** [comment]
That's extremely strange.

Apologies for not seeing this until now.

What firmware revision?

**Comment by jim13x** [comment]
I'm running RM520NGLAAR03A03M4G_01.201.01.201.  I had been having disconnect issues with RM520NGLAAR03A02M4GA, so I decided to try the newer firmware to see if that would help.  I'm going to reinstall it at the remote location this weekend to see how it goes.  

Thanks for supporting such a cool project!

**Comment by MickMorley** [comment]
@jim13x what firmware did you end of with and did your disconnects go away?

**Comment by jim13x** [comment]
> @jim13x what firmware did you end of with and did your disconnects go away?

I went back to RM520NGLAAR01A07M4G and it seems more stable.  No more PCIe linkdown errors.

**Comment by MickMorley** [comment]
Thank you @jim13x . There is information on https://github.com/iamromulan/quectel-rgmii-configuration-notes/discussions/11#discussioncomment-9806774, explaining that MR5210 isn't recommended anymore due to hardware issues on the board.

I removed tailscale because the simpleadmin and board would seem to freeze and not properly assign an IP address to my laptop (using CAT6 directly plugged in).  I only tried the latest RM520NGLAAR03A02M4GA but didn't see any PCIe linkdown errors when running `dmesg`.  I am going to try the new board that is recommended now.
